mms/serving_frontend.py

```diff
@@ -40,17 +38,14 @@ def __init__(self, app_name, batching=False, data_store_config=None):
         app_name : str
             App name to initialize request handler.
         """
-        def gen_prefix(prefix, length=5, chars=string.ascii_letters+string.digits):
-            return prefix + ''.join(random.choice(chars) for _ in range(length)) + '_'
-
         try:
             self.service_manager = ServiceManager()
             self.handler = FlaskRequestHandler(app_name)
 
             if batching:
-                # generate a pseudo-random prefix to avoid collisions between runs
-                prefix = gen_prefix(app_name)
-                data_store = DataStore(prefix, data_store_config)
+                # TODO generate a unique prefix in docker/mxnet-model-server and pass it into all gunicorn workers
```

Comment:
Good point to make a TODO. would this still be an issue after removing snapshotting?